modules/basics/src/main/resources/META-INF/com/opengamma/strata/config/base/FloatingRateNameData.ini

```diff
@@ -213,6 +213,9 @@ SGD-SOR-Telerate = SGD-SOR
 SGD-SOR-VWAP = SGD-SOR
 SGD-SOR-VWAP-Reference Banks = SGD-SOR
 
+# "THB-THOR/THB-THOR-OIS COMPOUND" is the ISDA 2021 code
```

Comment:
Are these both the standard codes, or is it just the one?

Reply:
both are